src/main/java/util/SimpleCommandLineParser.java

```diff
@@ -0,0 +1,88 @@
+package util;
```

Comment:
if you agree with doing a source format on the class above, this could use the same